backend/molgenis-emx2-tasks/src/main/java/org/molgenis/emx2/tasks/TaskServiceInDatabase.java

```diff
@@ -0,0 +1,332 @@
+package org.molgenis.emx2.tasks;
+
+import static org.molgenis.emx2.Column.column;
+import static org.molgenis.emx2.FilterBean.f;
+import static org.molgenis.emx2.FilterBean.or;
+import static org.molgenis.emx2.Operator.EQUALS;
+import static org.molgenis.emx2.Row.row;
+import static org.molgenis.emx2.TableMetadata.table;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.File;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.molgenis.emx2.*;
+import org.molgenis.emx2.sql.JWTgenerator;
+import org.molgenis.emx2.sql.SqlDatabase;
+
+public class TaskServiceInDatabase extends TaskServiceInMemory {
+  private Database database;
+  private String systemSchemaName = "ADMIN";
```

Comment:
I am happy to change this; SYSTEM sounds also good to me. I now choose admin to make clear this is for admins (not for systems).

Reply:
OK, I tried the change. However, as a data manager is it quite inconvenient to have to search to the 'SYSTEM' schema while 'ADMIN' is easy to find because it is listed on top. Therefore chose _SYSTEM_ as the name.